Assets/Scripts/Game/StatusChecker.cs

```diff
@@ -0,0 +1,70 @@
+using UnityEngine;
+
+public class StatusMonitor : MonoBehaviour
+{
+    public CardData energyZeroCard;
+    public CardData moneyZeroCard;
+    public CardData reputationZeroCard;
+
+    private ResourceManager resourceManager;
+
+    void Start()
+    {
+        resourceManager = FindObjectOfType<ResourceManager>();
+    }
+
+    void Update()
```

Comment:
Instead oh having everything in update maybe we should introduced a new method and call it CheckForStatusZero to encapsulate status checks, keeping Update clean and focused on just calling the method.

so do somthing like this? 

```  
void Update()
    {
        CheckForStatusZero();
    }

    private void CheckForStatusZero()
    {
        if (GameManager.MoneyStatus <= 0)
        {
            LoadSpecialEndingCard(moneyZeroCard);
            return;
        }

        if (GameManager.EnergyStatus <= 0)
        {
            LoadSpecialEndingCard(energyZeroCard);
            return;
        }

        if (GameManager.ReputationStatus <= 0)
        {
            LoadSpecialEndingCard(reputationZeroCard);
            return;
        }
    }
```